src/algebra/gcd_monoid.lean

```diff
@@ -820,4 +821,27 @@ gcd_monoid_of_lcm
   (λ a b c ac ab, normalize_dvd_iff.2 ((classical.some_spec (h c b) a).1 ⟨ac, ab⟩))
   (λ a b, normalize_idem _)
 
+/-- `ℕ` is a `gcd_monoid` -/
+instance : gcd_monoid ℕ :=
```

Comment:
@awainverse was this not already somewhere else? You know this part of the library best. Is this the correct place for the instance, or should it go somewhere in `data/nat/*`?

Reply:
It's been a minute, but it looks like the only global instances of `gcd_monoid` anywhere are `polynomial.gcd_monoid` and `int.gcd_monoid`. There is also `unique_factorization_monoid.to_gcd_monoid`, which is not an instance because it would cause a loop. Perhaps it's better if this just goes in `ring_theory/int/basic` next to `int.gcd_monoid`? I told @tb65536 it could go in either place.

Reply:
It seems weird to me to put this in `ring_theory/int/basic` when this doesn't use facts about the integers.

Reply:
(In case it's not clear, either place is actually still fine with me)

Reply:
I should mention that the `gcd_monoid` instance is also in #7180

Reply:
I'm happy to close #7180 but I'm wondering whether it makes more sense to deduce the `normalization_monoid` part from `unique (units nat)` which already exists - althought in the very weird `ring_theory/int/basic`. I think it would be better to move it to `data/nat/basic` where one can also find this fact (not as an instance).